### ISSUE TITLE
Hyperparameter optimization

### ISSUE THREAD

**Issue by JohanvandenHeuvel** [JohanvandenHeuvel/AggregationOfLocalExplanations#3]
- Optimize Learning rate: Seems very important according to RBM paper
- Is it possible to optimize momentum and regularization?
-
